File: PRAK605-2210817120013-Fathiah_Nuraisyah_Radam.c
#include <stdio.h>
int main(){
    int matrixA[50][50];
    int matrixB[50][50];
    int hasil [50][50];
    int baris, i, j, k, jumlah = 0;
    scanf("%d", &baris);
    printf("Matriks A\n");
    for (i = 0; i < baris; i++){
       for (j = 0; j < baris; j++){
            scanf("%d", &matrixA[i][j]);}}
    printf("Matriks B\n");
    for (i = 0; i < baris; i++){
       for (j = 0; j < baris; j++){
            scanf("%d", &matrixB[i][j]);}}
    for (i = 0; i < baris; i++){
      for (j = 0; j < baris; j++){
            for (k = 0; k < baris; k++){
                  jumlah = jumlah + matrixA[i][k] * matrixB[k][j];}
            hasil [i][j] = jumlah;
            jumlah = 0;}}
    printf("Matriks AXB\n");
      for (i = 0; i < baris; i++){
            for (j = 0; j < baris; j++){
                  printf("%d ", hasil[i][j]);}
            printf("\n");}
}

File: PRAK605-2210817120013-Fathiah_Nuraisyah_Radam.py
baris = int(input())
matrix = []
a=[]
b=[]
print("Matriks A")
for i in range(baris):
      v = [int (y) for y in input().split()]
      a.append(v)
      matrix.append(a)
print("Matriks B")
for i in range(baris):
      r = [int (y) for y in input().split()]
      b.append(r)
      matrix.append(b)
hasil=[[0 for i in range(baris)] for j in range(baris)]
for i in range(baris):
	for j in range(baris):
		for k in range(baris):
			hasil[i][j]=hasil[i][j]+a[i][k] * b[k][j]
print("Matriks AXB")
for i in range(baris):
      for j in range (baris):
            print(format(hasil[i][j]), end=" ")
      print()
